common/common-blackbox-setup/src/lib.rs

```diff
@@ -203,6 +204,18 @@ impl BaseSetup {
             });
     }
 
+    pub fn check_deposited_tokens_amount_is_empty(&mut self, tokens: Vec<TestTokenIdentifier>) {
```

Comment:
Empty is just if some token was never deposited.
I would say you should check for some value (can either be zero or some other value).
After deposit has some value, then after execute should be zero.